### DISCUSSION TITLE
ml 域名好像也挂了

### DISCUSSION THREAD

**Discussion by ZhReimu** [luolongfei/freenom#230] [General]
如图，前几天 dnspod 上看还是正常但无法解析，今天再看 dnspod 都开始告警了
![image](https://github.com/luolongfei/freenom/assets/58155247/03e5351e-d2d4-42a8-8015-09623cfe0f98)

**Comment by cyao2q** [comment]
一样问题 明明freenom后台显示 active正常 cloudflare无法正常解析了

**Comment by fxzxmic** [comment]
ml的相关通知详见nic.ml
